src/Cli.Tests/TestHelper.cs

```diff
@@ -1226,6 +1227,43 @@ public static Process ExecuteDabCommand(string command, string flags)
   }
 }
 ";
+        /// <summary>
+        /// Generates the config json string with the given depth limit in the form of json string.
+        /// example: { ""depth-limit"": 10 }
+        /// </summary>
+        /// <returns></returns>
+        public static string GenerateConfigWithGivenDepthLimit(string? depthLimitJson = null)
+        {
+            string depthLimitSection = depthLimitJson != null ? depthLimitJson : string.Empty;
+
+            string runtimeSection = $@"
+            ""runtime"": {{
+                ""rest"": {{
+                    ""path"": ""/api"",
+                    ""enabled"": true,
+                    ""request-body-strict"": true
+                }},
+                ""graphql"": {{
+                    ""path"": ""/graphql"",
+                    ""enabled"": true,
+                    ""allow-introspection"": true
```

Comment:
```suggestion
                    ""allow-introspection"": true,
```

needs comma?

Reply:
comma is added automatically if the {depthLimitSection} is not empty.